test/main.lua

```diff
@@ -203,6 +203,14 @@ local function clean_require(module)
     return require(module)
 end
 
+-- similar to clean_require above; forces clean load of scripts directly
+-- included from a test file. note that this does *not* force transitively
```

Comment:
I'm not sure quite how many levels you mean by "transitively" here, but I'm pretty sure that *any* script that calls `reqscript()` would not use our patched version here, because reqscript in its global environment would be coming from `BASE_G`: https://github.com/DFHack/dfhack/blob/dbd35359be68f98c0a8a0cd1fbf783235b3ee84a/library/lua/dfhack.lua#L657

Reply:
I'll clarify -- I meant that while scripts directly reqscript'd by the test file will be reloaded, scripts reqscript'd by the reqscript'd scripts would not.

Reply:
done